app/controllers/technical_metadata_controller.rb

```diff
@@ -22,7 +22,9 @@ def show_by_druid
 
   def file_infos
     params[:files].map do |file|
-      FileInfo.new(filepath: CGI.unescape(URI(file[:uri]).path), md5: file[:md5])
+      filepath = CGI.unescape(URI(file[:uri]).path)
+      filename = FilepathSupport.filename_for(filepath: filepath, basepath: params[:basepath])
```

Comment:
Now including the filename (filepath minus basepath) in addition to the complete filepath.